robosuite/eval/eval.py

```diff
@@ -0,0 +1,114 @@
+import os
+import h5py
+import mediapy as media
+import matplotlib.pyplot as plt
+import numpy as np
+import random
+import os
+import openai
+
+# openai.api_key = os.getenv("OPENAI_API_KEY")
+
+
+def load_hdf5(dataset_path: str) -> tuple:
+    if not os.path.isfile(dataset_path):
+        print(f"Dataset does not exist at \n{dataset_path}\n")
+        exit()
+
+    with h5py.File(dataset_path, "r") as root:
+        qpos = root["/observations/qpos"][()]
+        qvel = root["/observations/qvel"][()]
+        image_dict = dict()
+        for cam_name in root[f"/observations/images/"].keys():
+            image_dict[cam_name] = root[f"/observations/images/{cam_name}"][()]
+        action = root["/action"][()]
+
+    return qpos, qvel, action, image_dict
+
+
+def find_stopping_point(data: np.ndarray) -> int:
+    steps, _ = data.shape
+    for step in range(steps):
```

Comment:
nit: Feel free to ignore :)

Iterate from the end until we find the index where data[step] != 0